src/components/datepicker/datePicker.scss

```diff
@@ -165,21 +197,31 @@ md-datepicker[disabled] {
 
 // Open state for all of the elements of the picker.
 .md-datepicker-open {
+  overflow: hidden;
+
   .md-datepicker-input-container {
     @include rtl-prop(margin-left, margin-right, -$md-datepicker-button-gap);
 
     // The negative bottom margin prevents the content around the datepicker
     // from jumping when it gets opened.
     margin-bottom: -$md-datepicker-border-bottom-gap;
-    border: none;
+    border-color: #fff;
```

Comment:
Ah, it should be `transparent`, must've missed it while trying out stuff. I was trying to avoid it jumping by removing the border altogether.